CHANGELOG.md

```diff
@@ -1,71 +1,157 @@
-Changes since Idris 2 v0.3.0
-============================
+# Changelog
 
-Library changes:
-* Introduced `test` package.
-
-  - Moved `tests/Lib.idr` into new `test` package as `Test/Golden.idr`.
-
-  - Removed `contrib/Test/Golden.idr` which duplicated the test framework now in the `test` package.
-* Added `System.Console.GetOpt`,a library for specifying and parsing
-  command line options, to `contrib`.
-* Monad transformers in `Control.Monad` where restructured
-  and several new transformer types where added.
-* `Data.Colist` and `Data.Colist1` were added to `base`.
-* Add `Data.SnocList` to base and `data SnocList` to `Prelude.Types`.
-* `Data.Bits`, an interface for bitwise operations, was added to `base`.
-* Interfaces `Bifoldable` and `Bitraversable` were added to the `prelude`.
-* Interface `Data.Contravariant` for contravariant functors was added
-  to base.
-* `Control.Applicative.Const` was added to `base`.
-
-REPL/IDE mode changes:
-
-* Added `:search` command, which searches for functions by type
-* `:load`/`:l` and `:cd` commands now only accept paths surrounded by double quotes
+## Unreleased (idris2-next)
 
-Syntax changes:
+### Syntax changes
 
+* Desugar non-binding sequencing in do blocks to (`>>`)
+  ([#1095](https://github.com/idris-lang/idris2/pull/1095))
+* Multiline Strings with `"""` as delimiters
+  ([#1097](https://github.com/idris-lang/idris2/pull/1097))
+* Force strict identation after usage of `with` keyword
```

Comment:
```suggestion
* Force strict indentation after usage of `with` keyword
```

Reply:
Thanks for proof-reading / checking, committed. UwU